docs/introduction-to-smart-contracts.rst

```diff
@@ -72,9 +72,9 @@ Subcurrency Example
 
 The following contract will implement the simplest form of a
 cryptocurrency. It is possible to generate coins out of thin air, but
-only the person that created the contract will be able to do that (it is trivial
+only the person that created the contract will be able to do that (it is very easy
```

Comment:
I am personally (but also a common opinion in tech writer circles) to not use words like trivial, easy etc. For this text, I'd say remove the 'very' to lessen the strength of the sentence.